src/core/TooltipTableContent/index.tsx

```diff
@@ -1,19 +1,15 @@
-import { useTheme } from "@emotion/react";
 import Table from "@material-ui/core/Table";
 import TableBody from "@material-ui/core/TableBody";
-import TableCell from "@material-ui/core/TableCell";
 import TableContainer from "@material-ui/core/TableContainer";
 import TableRow from "@material-ui/core/TableRow";
-import cx from "classnames";
 import React from "react";
 import {
-  alertCss,
-  disabledCss,
+  Alert,
   ExtraProps,
-  rowLabelCss,
-  rowValueCss,
-  sectionCss,
-  sectionLabelCss,
+  RowLabel,
```

Comment:
I think MUI's recommendation is to use `@emotion/styled` to style components by default, and only use `@emotion/css` to generate css class names when we need to pass class names to an MUI's component's `classes` props!

https://emotion.sh/docs/styled

This also means we no longer need `classnames` package 🎉